includes/widgets/accordion.php

```diff
@@ -290,10 +290,7 @@ protected function _register_controls() {
 				'selectors' => [
 					'{{WRAPPER}} .elementor-accordion-icon, {{WRAPPER}} .elementor-accordion-title' => 'color: {{VALUE}};',
 				],
-				'scheme' => [
-					'type' => Schemes\Color::get_type(),
-					'value' => Schemes\Color::COLOR_1,
-				],
+				'global' => Global_Style::COLOR_PRIMARY,
```

Comment:
Please change the format to an array, here and all files.

Reply:
Done